docs/interpreter/jdbc.md

```diff
@@ -916,11 +912,15 @@ Dependencies
     <th>Excludes</th>
   </tr>
   <tr>
-    <td>org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.6.1-incubating</td>
+    <td>org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.9.0</td>
     <td></td>
   </tr>
   <tr>
-    <td>org.apache.hive:hive-jdbc:3.1.2</td>
+    <td>org.apache.hadoop:hadoop-client-api:3.3.6</td>
```

Comment:
There are several approaches(including Hadoop UGI) to use Kerberos with `kyuubi-hive-jdbc-shaded`, the current Zeppelin JDBC Interpreter only implements the UGI approach, so Hadoop deps are required. I can extend the Zeppelin code to support the JDK JAAS based Kerberos authentication to get rid of Hadoop deps in the future